paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -1377,6 +1364,8 @@ class PaparazziPluginTest {
   @Test
   fun accessibilityErrorsLogged() {
     val fixtureRoot = File("src/test/projects/validate-accessibility")
+    // this is only a warning message, so subsequent runs would otherwise be UP-TO-DATE
+    fixtureRoot.resolve("build").registerForDeletionOnExit()
```

Comment:
I see other instances of fixtureRoot.resolve being replaced with buildDir. Does that apply here?